x/supply/client/cli/query.go

```diff
@@ -0,0 +1,32 @@
+package cli
+
+import (
+	"github.com/ovrclk/akash/x/supply/query"
+	"github.com/ovrclk/akash/x/supply/types"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/spf13/cobra"
+)
+
+// GetCirculatingSupply returns circulation supply
+func GetCirculatingSupply(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "circulating",
```

Comment:
Since we are including all the supply information, may be we should change this to `all` or something better. @boz wdyt?

```suggestion
		Use:   "all",
```

Reply:
May be `summary` or `stats`

Reply:
:+1: summary